src/utils/utils.test.js

```diff
@@ -129,6 +129,33 @@ describe('getCourseUrlSlugPattern', () => {
       });
     },
   );
+
+  it(
+    'returns the old course url slug pattern when courseType is bootcamp and updatedSlugFlag is false',
+    () => {
+      const updatedSlugFlag = false;
+      const courseType = BOOTCAMP_SLUG;
+
+      expect(
+        utils.getCourseUrlSlugPattern(updatedSlugFlag, 'external-source', courseType),
+      ).toEqual({
+        slug_format: COURSE_URL_SLUG_PATTERN_OLD,
+        error_msg: 'Course URL slug contains lowercase letters, numbers, underscores, and dashes only.',
+      });
+    },
+  );
+
+  it(
+    'returns the bootcamp subdirectory slug pattern when courseType is bootcamp and updatedSlugFlag is true',
+    () => {
+      const updatedSlugFlag = true;
+      const courseType = BOOTCAMP_SLUG;
+
+      expect(
+        utils.getCourseUrlSlugPattern(updatedSlugFlag, 'external-source', courseType),
+      ).toEqual(JSON.parse(COURSE_URL_SLUGS_PATTERN)['external-source'][BOOTCAMP_SLUG]);
+    },
+  );
```

Comment:
You can use test.each to turn these into ddt style tests. 
Or remove updatedSlugFlag & courseType variables and use the values directly in fn call.

Reply:
done